README.md

```diff
@@ -76,10 +76,16 @@ building on versions below the ones stated. To install on a device,
 you **must** have a valid Apple Developer account license and have
 added the relevant mobile provisioning profiles from the Developer
 portal to your Xcode organizer (see Apple's website on instructions on
-how to do this).
+how to do this).  The profile should have these **exact** details, and 
+should include a distribution certificate:
```

Comment:
Distribution profiles are different from developer profiles. Unless someone is publishing to the App Store, they shouldn't need to create a distribution profile.

Reply:
Documentation updated.